packages/prettier-plugin-java/src/printers/prettier-builder.ts

```diff
@@ -2,8 +2,6 @@
 import { IToken } from "java-parser";
 import { doc } from "prettier";
 import Doc = doc.builders.Doc;
-import Concat = doc.builders.Concat;
-import Fill = doc.builders.Fill;
 
 const prettier = require("prettier").doc.builders;
```

Comment:
I think this  require() can be removed too. `doc` is imported, and it's not type only.